### DISCUSSION TITLE
文件关联一直为0%的客户端

### DISCUSSION THREAD

**Discussion by SteveYu000** [Simple-Tracker/qBittorrent-ClientBlocker#83] [General]
有点奇怪这个文件关联到底是什么，有个客户端一直在下载，但文件关联一直为0%，进度倒是在增加。我看我下载的时候会显示文件关联的百分比以及文件
![2024-06-23_015756](https://github.com/Simple-Tracker/qBittorrent-ClientBlocker/assets/111431635/2db20973-5ec9-4e7a-b0a7-667b5ddc7141)

**Comment by chierinyan** [comment]
文件关联指的是该peer拥有多少你需要的block. 对于已完成的种子0%是正常的.
